Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'govuk_taxonomy_helpers'
```

Comment:
I'm not sure actually, I can't find where this variable is used, but it's in the rails template Jenkinsfile, and gds-api-adapters (which I looked at as an example of another gem) also has it. 😕